### ISSUE TITLE
syntax error

### ISSUE THREAD

**Issue by Anthchirp** [xia2/i19#8]
Black formatting might be a nice idea, but in this instance it didn't work.
```
E     File "i19/command_line/screen.py", line 684
E       def _wilson_calculation(self):
E                                    ^
E   IndentationError: unindent does not match any outer indentation level
```

Could you also fix up the docstring indentations? ie. stuff like
```python
        """                                                                                                             
    TODO: Docstring                                                                                                     
    :param additional_parameters:                                                                                       
    :return:                                                                                                            
    """                                                                                                                 
```

**Comment by benjaminhwilliams** [comment]
Yep, sure thing.  I was perhaps a little rash to experiment with it on the last day of the year!

**Comment by benjaminhwilliams** [comment]
Fixed by b55a996bc172872982df1846bca0847df2959024
